From dbd5721bb8845fe408d53f48601df500d685065c Mon Sep 17 00:00:00 2001
From: Roman Yarullin <firefox883@gmail.com>
Date: Sun, 3 Nov 2019 01:20:04 +0200
Subject: [PATCH] sepolicy: Resolve storaged denials

*Label /sys/devices/msm_sdcc.1/mmc_host/mmc1/mmc1:0001/block/mmcblk0/stat as sysfs_disk_stat

* denied { read open } for name=stat dev=sysfs ino=19332 scontext=u:r:storaged:s0 tcontext=u:object_r:sysfs:s0 tclass=file permissive=1

* denied { getattr } for path=/sys/devices/msm_sdcc.1/mmc_host/mmc1/mmc1:0001/block/mmcblk0/stat dev=sysfs ino=19332 scontext=u:r:storaged:s0 tcontext=u:object_r:sysfs:s0 tclass=file permissive=1

Change-Id: Ic4047346cdc282568069b9abdb3d250c051d6579
---
 sepolicy/file_contexts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 66969cbd..4540ce06 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -157,6 +157,8 @@
 /sys/devices/bq51013b_wlc.77/power_supply/wireless(/.*)?                u:object_r:sysfs_batteryinfo:s0
 /sys/devices/battery_tm_ctrl.78/power_supply/batt_therm(/.*)?           u:object_r:sysfs_batteryinfo:s0
 
+/sys/devices/msm_sdcc.1/mmc_host/mmc1/mmc1:0001/block/mmcblk0/stat      u:object_r:sysfs_disk_stat:s0
+
 # Sysfs files used by qmuxd
 /sys/devices/virtual/smdpkt/smdcntl([0-9])+/open_timeout u:object_r:sysfs_smdcntl_open_timeout:s0
 
